manifests/v/VoyagerX/Vrew/0.8.17/VoyagerX.Vrew.installer.yaml

```
@@ -0,0 +1,20 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: VoyagerX.Vrew
+PackageVersion: 0.8.17
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
```

```suggestion
- silent
```
I know this is not your fault and its my YamlCreate.ps1 script's fault 🙄